internal/cmd/versions.go

```diff
@@ -0,0 +1,69 @@
+package cmd
+
+import (
+	"fmt"
+	"runtime/debug"
+	"strings"
+
+	"github.com/envoyproxy/gateway/internal/ir"
+	"github.com/spf13/cobra"
+)
+
+var (
+	// envOutput determines whether to output as environment settings
+	envOutput bool
+)
+
+// getVersionsCommand returns the server cobra command to be executed.
+func getVersionsCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "versions",
+		Aliases: []string{"version"},
+		Short:   "Show versions",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return versions()
+		},
+	}
+
+	cmd.PersistentFlags().BoolVarP(&envOutput, "env", "e", false,
+		"If set, output as environment variable settings.")
+
+	return cmd
+}
+
+// versions shows the versions of the Envoy Gateway.
+func versions() error {
```

Comment:
Globals are bad. Can we scope `envOutput`? Something like this:

```go
// getVersionsCommand returns the server cobra command to be executed.
func getVersionsCommand() *cobra.Command {
	// envOutput determines whether to output as environment settings
	var envOutput bool

	cmd := &cobra.Command{
		Use:     "versions",
		Aliases: []string{"version"},
		Short:   "Show versions",
		RunE: func(cmd *cobra.Command, args []string) error {
			return versions(envOutput)
		},
	}

	cmd.PersistentFlags().BoolVarP(&envOutput, "env", "e", false,
		"If set, output as environment variable settings.")e do this as

	return cmd
}

// versions shows the versions of the Envoy Gateway.
func versions(envOutput bool) error {
```
?

Reply:
Good point; done.